src/pages/Manager.vue

```diff
@@ -644,8 +644,20 @@ import CategoryFilterModal from '../components/modals/CategoryFilterModal.vue';
             }
         }
 
-        beforeCreate() {
+        async beforeCreate() {
             this.activeGame = GameManager.activeGame;
+
+            // Used by SearchAndSort, but need to be called here to
+            // ensure the settings are loaded before LocalModList
+            // accesses visibleModList from Vuex store.
+            await this.$store.dispatch('profile/loadOrderingSettings');
+
+            // Reset the mod list to prevent the previous profile's list
+            // flashing on the screen while a new profile's list is loaded.
```

Comment:
Does this actually solve it? The beforeCreate is async and I'm not sure if Vue even awaits for this to complete before rendering

Reply:
I'm under the assumption that everything that relies on this would get re-rendered if it changes regardless if there's a race condition or not, so it wouldn't be a problem?

Reply:
I tested this by making the Vuex dispatch function `sleep` a bit if an empty array of mods was passed to it. As a result, when called in `beforeCreate`, it rendered the proper mod list first, and then once the sleep finished, rendered the view with empty mod list. So a totally artificial test and quite unlikely that earlier call with empty array would complete later than a later call with non-empty array. But theoretically possible since the first call isn't awaited, methinks?